Week3/objectIncludingHyfTeachers.js

```diff
@@ -0,0 +1,7 @@
+// 13. Create an object that contains the teachers that you have had so far for the different modules.
+
+const hyf = new Object();
+
+hyf.Teachers = 'Philipp, Rob, Unmesh, Bonan, Yash';
```

Comment:
Ah, usually it's a good practice to use an `array` for `multiple-values`. So could you also use that here?